overlord/devicestate/handlers_systems.go

```diff
@@ -594,18 +595,17 @@ func (m *DeviceManager) doFinalizeTriedRecoverySystem(t *state.Task, _ *tomb.Tom
 	return nil
 }
 
-func recordSeededAndMarkRecoveryCapable(st *state.State, markCurrent bool, m *DeviceManager, model *asserts.Model, label string) error {
-	if markCurrent {
-		if err := m.recordSeededSystem(st, &seededSystem{
```

Comment:
newly created systems are no longer recorded as seeded?

Reply:
No, they aren't. They never should have been, this was from confusion on my behalf when looking at what happens during a remodel. They aren't considered seeded since the snaps that make up a newly created system do not actually have to be installed.